CMakeLists.txt

```diff
@@ -25,7 +25,10 @@ find_package(appdal REQUIRED)
 set(APPFWK_DEPENDENCIES ${CETLIB} ${CETLIB_EXCEPT} ers::ers logging::logging cmdlib::cmdlib rcif::rcif iomanager::iomanager opmonlib::opmonlib nlohmann_json::nlohmann_json pistache_shared
   coredal::coredal appdal::appdal oksdbinterfaces::oksdbinterfaces okssystem::okssystem)
 
-include_directories(${CMAKE_BINARY_DIR}/coredal ${CMAKE_BINARY_DIR}/appdal)
```

Comment:
Actually, I don't think it should be necessary to have these include_directories with the latest daq-cmake.

Reply:
Removed, still builds.